enclave_build/src/docker.rs

```diff
@@ -277,19 +277,19 @@ impl DockerUtil {
         // First try to find CMD parameters (together with potential ENV bindings)
         let act_cmd = async {
             match self.docker.images().get(&self.docker_image).inspect().await {
-                Ok(image) => Ok(image.config.cmd),
```

Comment:
could we also add an integration test for ENTRYPOINT and RUN to make sure we do not break this functionality in the future ?

Reply:
yes, this commit seems to have added support for it.
https://github.com/aws/aws-nitro-enclaves-cli/commit/d5b7f16e7575c3acc2b4c14ac45e470db13eaf8f

Reply:
Added a test using nitro-cli build-enclave command.